iphone/Classes/Base64Transcoder.c

```diff
@@ -129,7 +129,7 @@ bool Base64EncodeData(const void *inInputData, size_t inInputDataSize, char *out
 		outOutputData[theOutIndex++] = kBase64EncodeTable[(theInPtr[theInIndex] & kBits_00000011) << 4 | (theInPtr[theInIndex + 1] & kBits_11110000) >> 4];
 		outOutputData[theOutIndex++] = kBase64EncodeTable[(theInPtr[theInIndex + 1] & kBits_00001111) << 2 | (theInPtr[theInIndex + 2] & kBits_11000000) >> 6];
 		outOutputData[theOutIndex++] = kBase64EncodeTable[(theInPtr[theInIndex + 2] & kBits_00111111) >> 0];
-		if (theOutIndex % 74 == 72)
+		if (theOutIndex % 76 == 74)
```

Comment:
I'm going to go out on a limb here and assume the number 76 came from this: https://lists.w3.org/Archives/Public/w3c-ietf-xmldsig/2001AprJun/0183.html

Sounds like the spec requires lines no longer than 76 characters for Base64 encoding?

Looking further, that is *sometimes* true: 
- https://tools.ietf.org/html/rfc4648#page-3
- https://en.wikipedia.org/wiki/Base64

Given that we don't match between Android and iOS/Windows, we should come up with what we think is a reasonable default imply regard line newlines/line length. Windows does what iOS does right now, BTW.

I'd argue that our base64 encoder method should actually take an options argument to control this behavior. Sort of like Android's API does: https://developer.android.com/reference/android/util/Base64.html